src/styleguide/routes/picture.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <l-default>
+    <e-picture alt="Altext"
+               width="300"
+               height="423"
+               :sizes="sizes"
+               srcset="https://images.pexels.com/photos/10313905/pexels-photo-10313905.jpeg"
+               fallback="https://images.pexels.com/photos/9074921/pexels-photo-9074921.jpeg"
+    />
+  </l-default>
+</template>
+
+<script>
+  import ePicture from '@/elements/e-picture.vue';
+  import LDefault from '../../layouts/l-default.vue';
```

Comment:
todo: please use `camelCase` and update usage.

```suggestion
  import lDefault from '../../layouts/l-default.vue';
```